README.md

```diff
@@ -3,14 +3,17 @@
   <a href="https://goreportcard.com/report/github.com/spicetify/spicetify-cli"><img src="https://goreportcard.com/badge/github.com/spicetify/spicetify-cli"></a>
   <a href="https://github.com/spicetify/spicetify-cli/releases/latest"><img src="https://img.shields.io/github/release/spicetify/spicetify-cli/all.svg?colorB=97CA00?label=version"></a>
   <a href="https://github.com/spicetify/spicetify-cli/releases"><img src="https://img.shields.io/github/downloads/spicetify/spicetify-cli/total.svg?colorB=97CA00"></a>
-  <a href="https://discord.gg/VnevqPp2Rr"><img src="https://img.shields.io/discord/842219447716151306?label=Chat&logo=discord&logoColor=discord"></a>
-  <a href="https://www.reddit.com/r/spicetify"><img src="https://img.shields.io/reddit/subreddit-subscribers/spicetify?style=social"></a>
+  <a href="https://discord.gg/VnevqPp2Rr"><img src="https://img.shields.io/discord/842219447716151306?label=chat&logo=discord&logoColor=discord"></a>
+  <a href="https://www.reddit.com/r/spicetify"><img src="https://img.shields.io/reddit/subreddit-subscribers/spicetify?logo=reddit"></a>
 </p>
 
+---
+
+
 Command-line tool to customize the official Spotify client.
 Supports Windows, MacOS and Linux.
 
-<img src="https://user-images.githubusercontent.com/26436809/118751529-d0abcf00-b8a4-11eb-9876-8b15f930a691.png" alt="img" align="right" width="500px">  
+<img src="https://i.imgur.com/VUXGKYZ.png" alt="img" align="right" width="560px" height="400px">  
```

Comment:
Maybe we should upload the image to the github repo, in case the link goes down in the future?

Reply:
Yeah sure, do upload it in the repo if you have the means to

Reply:
You can do it in this PR

Reply:
In my repo?

Reply:
Done doing it. /.github/assets/logo.png

Reply:
Added to README.